### ISSUE TITLE
[Suggestion] Upgrade for only exporting excess items

### ISSUE THREAD

**Issue by neoc666** [jaquadro/StorageDrawers#468]
I'm using chained compacting drawers and cobblestone generators in my SF3 world to generate all compressed versions of cobblestone. It would be nice to have a drawer upgrade which would expose only items above a certain limit (say up to max stacks - 1) to (Ender-) conduits so I could have one drawer with at least one stack of basic cobblestone available at all times. Or maybe there is a straightforward solution to this I just don't know?

**Comment by codewarrior0** [comment]
There is! Use a drawer controller. When multiple drawers are locked to the same item, it will prefer to put incoming items into the nearest drawer. I've used it with two Redstone drawers, to send any Redstone above a certain limit (40 stacks) into a Centrifuge to make Ruby Dust.

**Comment by neoc666** [comment]
That might work - thanks!

**Comment by codewarrior0** [comment]
It definitely works. I just used it for the same problem you described: I connected my cobble generator to a controller slave, put a cobble drawer close to the drawer controller, and the chain of cobble compacting drawers further away from the controller. It filled up the first drawer before starting to fill the compacting chain.

**Comment by neoc666** [comment]
I had another idea later: You can use priorities on Ender IO item conduits. Connect two destination chests via the same pipe and set the priority on one receiving end higher than on the other. Tested manually with 3 chests and a stack of cobble, only the high priority chest received items.
